contracts/issuer/lib/IIssuerEvents.sol

```diff
@@ -30,4 +30,9 @@ interface IIssuerEvents {
 
   event MemberAdded(address indexed member);
   event MemberRemoved(address indexed member);
+
+  // Governors.
+
+  event GovernorRemovedFromFast(address indexed fast, address indexed governor);
+  event GovernorAddedToFast(address indexed fast, address indexed governor);
```

Comment:
I've added `indexed` on both of these parameters as you might want to filter differently depending on the use case. The underlying `GovernorAdded` etc events have a single parameter which is the member's address.